app/src/main/res/layout/fragment_home_screen.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    tools:context=".ui.overview.OverviewFragment">
+
+    <data class=".OverviewBinding">
+
+        <variable
```

Comment:
Do you use this?

Reply:
yeah, I use it , but  it's just used for naming purposes. 
I can get rid of it.. but can I leave it like this?

Reply:
I don't really understand. I was asking if you use the viewModel variable since it doesn't seem to be used in the xml. Do I miss something?